lib/datadog/security/configuration.rb

```diff
@@ -0,0 +1,38 @@
+require 'datadog/security/configuration/settings'
+
+module Datadog
+  module Security
+    module Configuration
```

Comment:
Was the existing `ddtrace` Configuration file too tracing-centric?

Reply:
Not really but I could not make use of it without having either the tracer loaded or the configuration having some global singleton-ish thing and both configs being mixed up.

Reply:
Confirmed through some chat that the current state of configuration is too global yet (especially with contrib integrations), although that is bound to improve with time on Core.